Xamarin.Forms.Core/ContextItem.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Windows.Input;
+
+namespace Xamarin.Forms
+{
+	public abstract class ContextItem : ContentView
```

Comment:
I would reuse MenuItem, and not allow a full ContentView as swipe item. Else we will end up with full pages being rendered for swipe, killing performances.

If a full swipeView with ContentView is required, CarouselView should be used.

Reply:
Some discussion around this one https://xamarinhq.slack.com/archives/CF98LQ1L3/p1568666466065500 and the reasons for using a view

Reply:
Ok, now (with the latest changes) we are not using Views. Actually, on all platforms the SwipeItems are created when you start the Swipe and are disposed when you close the SwipeView.
I have reviewed designs (Dribbble, etc.) and I think there are cases where a View fits better. I will update the Spec with some ideas. Maybe we can have a first Release without allowing Views and if it is something demanded we can add support later.

What do you think?. BTW, I have a small prototype adding support to Views and include it would not be complex, if you see it important we can add it in this PR.